oam/install_scripts/post-install

```diff
@@ -93,53 +95,65 @@ if [ -f libstdc++.so.6.0.14 ]; then
 fi
 
 # softlink for libperl.sp, used by cplogger
-$SUDO ln -s /usr/lib64/perl5/CORE/libperl.so /usr/lib64/libperl.so >/dev/null 2>&1
-
-#setup profile files
-profileFileAlias=/etc/profile.d/columnstoreAlias.sh
-profileFileEnv=/etc/profile.d/columnstoreEnv.sh
+ln -s /usr/lib64/perl5/CORE/libperl.so /usr/lib64/libperl.so >/dev/null 2>&1
 
 if [ $installdir != "/usr/local/mariadb/columnstore" ]; then
         sed -i -e s@/usr/local/mariadb/columnstore@$installdir@g $installdir/bin/columnstoreAlias
 fi
 
 if [ $user != "root" ]; then
-	sudo rm -f $profileFileEnv
-	sudo rm -f $profileFileAlias
-	
-	sudo touch $profileFileEnv
-	sudo chmod 666 $profileFileEnv
-	egrep -qs 'MariaDB Columnstore Non-Root' ${profileFileEnv}
+	if [ -f $prefix/.bash_profile ]; then		
+		profileFile=$prefix/.bash_profile
+	elif [ -f $prefix/.profile ]; then		
+		profileFile=$prefix/.profile
+	else
+		profileFile=$prefix/.bashrc
+		touch $prefix/.bashrc
+	fi
+		
+	if [ ! -f ${profileFile}_backup ]; then
+	    cp $profileFile ${profileFile}_backup
+	fi
```

Comment:
If I were the user, I'd by annoyed that something just messed with my login scripts.  Better to tell them what to do and let them decide to make these mods.

Reply:
A second reading makes me think this code is actually referring to /usr/local/.bash_profile (and friends), which I don't think is what is intended.  Isn't $prefix /usr/local here?
Disclaimer:  It's getting late and I might be reading this wrong.